kotlin-react-router-dom/src/main/generated/react/router/dom/URLSearchParamsInit.kt

```diff
@@ -0,0 +1,5 @@
+// Automatically generated - do not modify!
+
+package react.router.dom
+
+typealias URLSearchParamsInit = String // string | ParamKeyValuePair[] | Record<string, string | string[]> | URLSearchParams
```

Comment:
I think signature `Record<string, string[]>` is most common and useful (not `String`)